tests/impl.cpp

```diff
@@ -7587,7 +7587,17 @@ result_t test_mm_round_sd(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_round_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const float *_a = impl.mTestFloatPointer1;
+    const float *_b = impl.mTestFloatPointer1;
+    float dx = roundf(_b[0]);
+    float dy = _a[1];
+    float dz = _a[2];
+    float dw = _a[3];
+
+    __m128 a = do_mm_load_ps(_a);
+    __m128 b = do_mm_load_ps(_b);
+    __m128 c = _mm_round_ss(a, b, _MM_FROUND_CUR_DIRECTION);
+    return validateFloatError(c, dx, dy, dz, dw, 0.001f);
```

Comment:
The error is not necessary.
We can simply use `validateFloat` function.

Reply:
If I use `validateFloat()` we can't even pass `make check`

Reply:
However, I think rounding should not contain any error.

Reply:
I saw the return value of `_mm_round_ps()` contains error, too.

https://github.com/DLTcollab/sse2neon/blob/master/tests/impl.cpp#L7723

Reply:
Should I open another PR to solve them together?

Reply:
You can fix this pull request first and then open another pull request for `_mm_round_ps()`.

Reply:
Something weird I found is `0.5` is round to `0` instead of `1`, but other numbers round successfully, e.g. `0.6` to `1.0`

Reply:
`0.5` round to `0` might happen since floating-point rounding has 5 different mode.
Please check https://en.wikipedia.org/wiki/Floating-point_arithmetic#Rounding_modes for reference.

Reply:
done